rdflib/graph.py

```diff
@@ -847,26 +847,32 @@ def set(
     def subjects(
         self,
         predicate: Union[None, Path, _PredicateType] = None,
```

Comment:
If `object` gets a plurality expansion, why not `predicate` too (and likewise with other functions)?  Something to do with `Path`?